frontend/src/app/main/ui/dashboard/team.cljs

```diff
@@ -108,7 +116,9 @@
           (let [params (:clean-data @form)
                 mdata  {:on-success (partial on-success form)
                         :on-error   (partial on-error form)}]
-            (st/emit! (dd/invite-team-member (with-meta params mdata)))))]
+            (st/emit! (dd/invite-team-member (with-meta params mdata)))
+(st/emit! (dd/fetch-team-invitations))
```

Comment:
looks not indented correctly
also: a single st/emit! can be used to emit the two events.